src/utils/map-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { Map as MapBoxMap } from 'mapbox-gl';
+import { LayerKey, BoundaryLayerProps } from '../config/types';
+import { getBoundaryLayers } from '../config/utils';
+
+export function isLayerOnView(map: MapBoxMap | undefined, layerId: LayerKey) {
+  return map
+    ?.getStyle()
+    .layers?.map(l => l.source)
+    .includes(`layer-${layerId}`);
+}
+
+export function boundaryOnView(map: MapBoxMap | undefined): BoundaryLayerProps {
```

Comment:
Add a docstring for this function.
Also, shouldn't this be named: `boundariesOnView` instead?

Reply:
Yeah. Better have `boundariesOnView` instead of picking just one. Especially that this util function will definitely be used somewhere else. Thus, just getting one was limiting